tables/output_tables.Rmd

```diff
@@ -149,15 +164,6 @@ soi_df <- tmb_hist %>%
       sample_id == "7316-2189" ~ "Lynch Syndrome",
       Kids_First_Biospecimen_ID == "BS_85Q5P8GF" ~ "Lynch syndrome",
```

Comment:
I just noticed that the capitalization of "Lynch syndrome" here is inconsistent.  Also, can we be consistent about the id we use here? Presumably `sample_id` is fine for all of these.

Finally, elsewhere, there now appear to be no samples with only one predisposition, so we can probably remove the `(s)` from the header.

Reply:
Sure, yeah I think I may not have used sample_id for all since there are some diffs between T/N sample ids and it was hard for me to keep track, but I think better to just use patient ID here, since it is a germline predisposition and it is patient-level, so I can update.

Reply:
Ah, I think I had `7316-515` as Other, NOS since that path report did not have a specific syndrome listed, but then another sample from the same pt `7316-2085` did have Lynch annotated. Since this is patient, not tumor-level, I updated this to match for the same patient.